.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,70 @@
+name: Bug Report
+description: Something isn't working as expected
+labels: [bug]
+body:
+- type: markdown
+  attributes:
+    value: "## Thank you for contributing to Swift-DocC!"
+- type: markdown
+  attributes:
+    value: |
+      Before you submit your issue, please complete each textarea
+      below with the relevant details for your bug, and complete
+      the steps in the checklist
+- type: textarea
+  attributes:
+    label: Description
+    description: |
+      A short description of the incorrect behavior. 
+      If you think this issue has been recently introduced and did not occur in an 
+      earlier version, please note that. If possible, include the last version that 
+      the behavior was correct in addition to your current version.
+  validations:
+    required: true
+- type: checkboxes
+  attributes:
+    label: Checklist
+    options:
+    - label: If possible, I've reproduced the issue using the `main` branch of this package.
+      required: false
+    - label: This issue hasn't been addressed in an [existing GitHub issue](https://github.com/apple/swift-docc/issues).
+      required: true
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: Describe what you expected to happen.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Actual behavior
+    description: Describe or copy/paste the behavior you observe.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Steps To Reproduce
+    description: |
+      Explanation of how to reproduce the incorrect behavior. 
+      This could include a reduced version of your documentation bundle,
+      or a link to the documentation (or code) that is exhibiting the issue.
+    placeholder: |
+      1. In this environment...
+      2. With this config...
+      3. Run '...'
+      4. See error...
```

Comment:
Is there a way to have GitHub play nice with the line breaks here?

<img width="619" alt="Screen Shot 2022-07-01 at 10 21 14 AM" src="https://user-images.githubusercontent.com/6750147/176941392-bb2b4aef-99cf-4913-afbf-bd1d7cca594a.png">

Reply:
We can use a separate line to break it. Check the result here https://github.com/Kyle-Ye/Issue/issues/new?assignees=&labels=bug&template=bug_report.yml